client/components/mma/accountoverview/manageProducts/ManageProductV2.tsx

```diff
@@ -244,31 +257,23 @@ const InnerContent = ({
 								Update payment method
 							</LinkButton>
 						)}
-
-					<div
-						css={css`
-							margin-left: ${space[5]}px;
-						`}
-					>
-						{!hasCancellationPending && (
-							<Button
-								priority="subdued"
-								onClick={() => {
-									navigate(
-										'/cancel/' +
-											specificProductType.urlPart,
-										{
-											state: {
-												productDetail: productDetail,
-											},
-										},
-									);
-								}}
+					{!hasCancellationPending && !isSelfServeCancellationAllowed && (
```

Comment:
I think this should be `isSelfServeCancellationAllowed` rather?

Reply:
also seems like we need an alternative for if selfServiceCancellation isn't allowed. Not sure who should make the design call for that

Reply:
Updated, as suggested.